strawberryfields/cli/__init__.py

```diff
@@ -0,0 +1,311 @@
+
+    Returns:
+        ArgumentParser: an argument parser object that defines the related
+        options
+    """
+    parser = argparse.ArgumentParser(
+        usage="starship <command> [<args>]",
+        description="These are common options when working on the Xanadu cloud platform.",
+    )
+
+    # Setting a title for the general options (requires setting a private
+    # attribute)
+    parser._optionals.title = "General Options"
+
+    # Adding the pinging general option
+    parser.add_argument(
+        "--ping", "-p", action="store_true", help="Tests the connection to the remote backend."
+    )
+
+    # Adding subparsers configure and input
+    subparsers = parser.add_subparsers(title="Commands")
+
+    # Adding the configure subparser
+    configure_parser = subparsers.add_parser(
+        "configure", help="Configure each detail of the API connection."
+    )
+    configure_parser.set_defaults(func=configure)
+
+    configure_parser.add_argument(
+        "--token",
+        "-t",
+        type=str,
+        help="Configure Strawberry Fields with your Xanadu cloud platform API token.",
+    )
+    configure_parser.add_argument(
+        "--local", "-l", action="store_true", help="Create a local configuration file in the current directory."
+    )
+
+    # Adding the input subparser
+    run_parser = subparsers.add_parser("run", help="Run a blackbird script.")
+    run_parser.add_argument(
+        "input", type=str, help="The input blackbird script to run.",
+    )
+    run_parser.set_defaults(func=run_blackbird_script)
+    run_parser.add_argument(
+        "--output",
+        "-o",
+        help="Path to the output file, where the results of the program will be written (stdout by default).",
+    )
+
+    return parser
+
+
+def configure(args):
+    r"""An auxiliary function for configuring the API connection to the Xanadu
+    cloud platform.
+
+    Can be used to simply save the authentication token with default
+    configuration options. Alternatively, a wizard is provided for full
+    configurability.
+
+    **Example:**
+
+    The following is the easiest way to configure the connection to the cloud platform.
+
+    Here we show how the `starship configure --token MYAUTH` command can create a
+    configuration. It is worth noting that `MYAUTH` stands for the
+    authentication token. Once done, we have a look at the newly created
+    configuration file.
+
+    Example run in a Linux based operating system:
+
+    .. code-block:: bash
+
+        $starship configure --token MYAUTH
+        $cat  ~/.config/strawberryfields/config.toml
+        [api]
+        authentication_token = "MYAUTH"
+        hostname = "platform.strawberryfields.ai"
+        use_ssl = true
+        port = 443
+
+    The same example run on a Windows operating system:
+
+    .. code-block:: powershell
+
+        $starship configure --token MYAUTH
+        $cat C:\Users\USERNAME\AppData\Local\Xanadu\strawberryfields\config.toml
```

Comment:
Awesome to also have a windows example! I didn't know `cat` worked in powershell 👀 
```suggestion
        $ starship configure --token MYAUTH
        $ cat C:\Users\USERNAME\AppData\Local\Xanadu\strawberryfields\config.toml
```